scripts/flattener_mods/gather.py

```diff
@@ -305,6 +318,9 @@ def gather_pooled_metadata(obj_type, properties, values_to_add, objs, connection
 					if v == 'NCIT:C17998':
 						v = 'unknown'
 					value.append(v)
```

Comment:
This line might need to be removed? It appends v to value, and then later on that same v is appended to value again in the else statement.

Reply:
Yes! Thank you for catching that! Let's drop the append()